src/index.html

```diff
@@ -3,11 +3,667 @@
+                action="#"
+                metod="get"
+                class="send-us-a-message__form"
+              >
+                <input
+                  name="email"
+                  type="email"
+                  placeholder="Your email"
+                  required
+                  class="send-us-a-message__field"
+                >
+
+                <input
+                  name="name"
+                  type="text"
+                  placeholder="Your name"
+                  required
+                  class="send-us-a-message__field"
+                >
+
+                <textarea
+                  name="message"
+                  rows="3"
+                  placeholder="Your message"
+                  required
+                  class="send-us-a-message__field send-us-a-message__field--textarea"
+                ></textarea>
+
+                <button type="submit" class="send-us-a-message__button button">
+                  Send
+                </button>
+              </form>
+            </div>
+
+            <div class="contact-us__send-us
+              grid__item
+              grid__item--tablet-1-4
+              grid__item--small-desktop-7-12
+              grid__item--desktop-8-12
+              grid__item--large-screen-8-12
+            ">
+              <h2 class="contact-us__title">
+                Contact us
+              </h2>
+
+              <div class="contact-us__wrap-link">
+                <h3 class="contact-us__subtitle">
+                  Call us
+                </h3>
+
+                <a href="tel:654 321 987" class="contact-us__link">
+                  654 321 987
+                </a>
+              </div>
+
+              <div class="contact-us__wrap-link">
+                <h3 class="contact-us__subtitle">
+                  Visit us
+                </h3>
+
+                <a
+                  href="https://goo.gl/maps/z3YcZPBaied7qBEN9"
+                  target="_blank"
+                  class="contact-us__link--pink"
+                >
+                  2905 West Drive, Buffalo Grove
+                </a>
+              </div>
+
+              <div class="contact-us__wrap-link">
+                <h3 class="contact-us__subtitle">
+                  Our socials
+                </h3>
+
+                <div class="contact-us__wrap-socials">
+                  <a
+                    href="https://www.facebook.com/"
+                    class="contact-us__social contact-us__social--fb"
+                    target="_blank"
+                  ></a>
+                  <a
+                    href="https://twitter.com/"
+                    class="contact-us__social contact-us__social--tw"
+                    target="_blank"
+                  ></a>
+                  <a
+                    href="https://www.instagram.com/"
+                    class="contact-us__social contact-us__social--inst"
+                    target="_blank"
+                  ></a>
+                </div>
```

Comment:
Again here, get rid of such nesting, elements like nav, section etc. alone server as containers ,
![image](https://github.com/mate-academy/layout_dia/assets/87986955/6f36c25e-f3d3-459e-b3ff-5f0e68d43d7a)